assets/js/pages/SapSystemDetails/tableConfigs.jsx

```diff
@@ -79,7 +79,7 @@ export const getSystemInstancesTableConfiguration = ({
     {
       title: '',
       key: 'actions',
-      className: 'w-40',
```

Comment:
This file changes align instance operations button and site operation button in the same horizontal position, closer to the right margin